main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAttributionSchema.java

```diff
@@ -0,0 +1,39 @@
+package org.mobilitydata.gtfsvalidator.table;
```

Comment:
So looking at this more given that the `main` depends on `processor` to auto-generate code, I guess it's debatable how useful the table interfaces are without the processor.

Reply:
Unfortunately, this will lead to a circular dependency:
* `processor` depends on classes from `core` (`GtfsTable`, `ForeignKey`, `ForeignKeyError`)
* `core` depends on `processor` to generate classes from GTFS schema.

Here is what Gradle says:

```
:core:classes
\--- :core:compileJava
     +--- :core:jar
     |    \--- :core:classes (*)
     \--- :processor:jar
          \--- :processor:classes
               \--- :processor:compileJava
                    \--- :core:jar (*)

(*) - details omitted (listed previously)
```

> I think it would be useful to have the core GTFS schema definition in the core package so it could be used as a dependency in another project.

I can split `main` in two parts:
* `schemastd` - definition of the standard GTFS schema + specific validations;
* `cli` - the `org.mobilitydata.gtfsvalidator.cli` package 

What do you think?

Reply:
>I can split main in two parts:
>
>* schemastd - definition of the standard GTFS schema + specific validations;
* cli - the org.mobilitydata.gtfsvalidator.cli package
>
>What do you think?

After looking at this further I could go either way on this. 

In the interest of development simplicity I think fewer modules are better. And, looking at it from a dependency-perspective, I don't think the CLI dependencies would really add much to a JAR. 

Alternately, if we want to have a strict separation between "application"-level classes and the data model it could make sense to split them.

@aababilov You have to the best handle on the code as-is so I'd defer to your opinion.

Reply:
I agree, the CLI is currently so tiny right now that it does not make the difference. Let's stay with the current `main` subproject and we will split it later if needed.